wasabi/research-club/cjdns.md

```diff
@@ -109,215 +114,208 @@ And what they do is they manage all of the buying and selling of bandwidth lease
 
 ## Limitations to Distributed Hash Table based routing model. / Solving the problem of: "my internet doesn't work? Who do I call?"
 
-Speaker 1: 00:09:30
```

Comment:
We need the timestamps after a chapter title, even if it's the same speaker as before the chapter title. In this case it makes sense to move this sentence above the chapter as it's part of the previous chapter.